index.html

```diff
@@ -23,6 +26,10 @@
 		<img class="desc-image" src="{{ "images/manipulator.png" | relative }} " />
 		<div class="desc-text">
 			<p>
+				<!-- 
+				You might want to explain a bit more what modules are, before describing what they do.
+				Something like "Plethora adds items caled Modules.  Modules expose additional functionality to computers.  For example, the block scanner allows a non-command computer to retrieve information about surrounding blocks."
+				-->
 				Modules expose additional functionality to computers, such as the block scanner which will tell you all
 				about the nearby blocks or the introspection module which allows computers to interface with the
 				player's inventory.
```

Comment:
> I mean, you could always pipe them off to a turtle and craft there which is what Artist will end up doing.

To clarify on this point: this would only work if you have CCTweaks installed as that is what allows transferring items across the network.